index.html

```diff
@@ -73,98 +73,54 @@ <h2>Glad to see you!</h2>
-                        </div>
-                        <p class="card-descript">A daily selection of privately 
-                            personalized reads; no accounts or sign-ups required.
-                        </p>
-                        <ul class="languages">
+                    <img src="./images/Portfolio-popup.png" alt="my first work">
+                </div>
+                <div class="window-details">
+                    <p class="window-descript">
+                        Lorem Ipsum is simply dummy text of the printing
+                         and typesetting industry. Lorem Ipsum has been the
+                          industry's standard dummy text ever since the 1500s, 
+                          when an unknown printer took a galley of type and 
+                          scrambled it 1960s with the releaLorem Ipsum is simply dummy text 
+                          of the printing and typesetting industry. Lorem Ipsum has been the
+                           industry's standard dummy text ever since the 1500s, when 
+                              a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled 
+                              it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting 
+                              industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown
+                               printer took 
+                        a galley of type and scrambled it 1960s with the relea
+                    </p>
+                    <div class="aside-clm">
+                        <ul class="window-languages">
                             <li>HTML</li>
                             <li>CSS</li>
                             <li>Javascript</li>
+                            <li>bootstrap</li>
+                            <li>Ruby</li>
+                            <li>github</li>
                         </ul>
-                        <button>See Project</button>
-                    </div>
-                </div>                                                                                          
-                <div class="card card-4">
-                    <img src="./images/Snapshoot-Portfolio-4.svg" alt="my first work">
-                    <div class="details">
-                        <h2>Multi-Post Stories</h2>
-                        <div class="canopy">
-                            <h4>Canopy</h4>
-                            <ul>
-                                <li>Back End Dev</li>
-                                <li>2015</li>
-                            </ul>
+                        <div class="window-btn-links">
+                            <button>See live <img src="./images/Icon-live.svg" alt="see live icon"></button>
+                            <button>See source <img src="./images/Vector-seeGit.svg" alt="see live icon"> </button>
```

Comment:
- [ ] On hover the social media icon is disappearing,

![image](https://user-images.githubusercontent.com/77038610/146387755-f3f65a19-06ee-45a3-a875-4201bc1c4c79.png) 

I recommend you to set the color of the icon to white on hover 😉